mlflow/R/mlflow/Dockerfile.dev

```diff
@@ -0,0 +1,12 @@
+FROM rocker/r-ver:4.1.2
+
+WORKDIR /mlflow/mlflow/R/mlflow
+RUN apt-get update -y
+RUN apt-get install git wget libxml2-dev libgit2-dev -y
+RUN TEMP_DEB=$(mktemp) && \
+    wget --directory-prefix $TEMP_DEB https://github.com/jgm/pandoc/releases/download/2.16.2/pandoc-2.16.2-1-amd64.deb && \
+    dpkg --install $(find $TEMP_DEB -name '*.deb') && \
```

Comment:
Can we install pandoc by `RUN apt-get install pandoc -y` ?

Reply:
The version installed by pandoc is 2.5 which is too old and contains a bug.

Reply:
I inserted a comment.